Front-End/challenge1/script.js

```diff
@@ -1 +1,15 @@
 // Modify this file only
+// Modify this file only
+document.getElementById('decrease').addEventListener('click', function() {
+    let counter = document.getElementById('counter');
+    let currentDecrease = parseInt(counter.textContent);
+    counter.textContent = currentDecrease - 1;
+    currentDecrease <=0 ? alert('Numeros  negativos negativos'): null;
```

Comment:
I think the validation must follow a different order in the execution to work correctly.